go.mod

```diff
@@ -57,6 +57,7 @@ require (
 	bitbucket.org/creachadair/shell v0.0.7 // indirect
 	cloud.google.com/go/compute v1.18.0 // indirect
 	cloud.google.com/go/compute/metadata v0.2.3 // indirect
+	cuelang.org/go v0.4.3 // indirect
```

Comment:
What did we change that added all of these libraries?

Reply:
I was playing around with something in an early commit and had a merge conflict when I merged main back into this branch. After merging main back i ran a `go mod tidy` which is where this came from.

Reply:
It appears these dependency updates are necessary for some of the functionality we are now doing with cosign. I tried to copy over the go.mod and go.sum from the current main branch and couldn't get a build to resolve.